sklearn/linear_model/_ransac.py

```diff
@@ -89,6 +89,9 @@ class RANSACRegressor(
     RANSAC is an iterative algorithm for the robust estimation of parameters
     from a subset of inliers from the complete data set.
 
+    For a better understanding see example,
```

Comment:
I would put this at the end of the "Examples" section and write "For a more detailed example, see ..."